src/schemes/schemes.jl

```diff
@@ -1,9 +1,6 @@
-# Include open boundary system first,
-# since this system should not depend on specific systems.
-include("boundary/open_boundary/system.jl")
-
 # Include all schemes without rhs first. The rhs depends on the systems to define
 # interactions between the different system types.
```

Comment:
```suggestion
# interactions between the different system types.
# Viscosity requires the open boundary system.
```
Or is it only the viscosity? Could you just include the viscosity later where the rhs is included? After `Include rhs for all schemes`.